src/server/csp.ts

```diff
@@ -137,6 +137,9 @@ function cspEditions(
 			? 'https://platform.twitter.com https://cdn.syndication.twimg.com'
 			: ''
 	};
+	frame-src https://www.theguardian.com ${
```

Comment:
You may want to add this to the existing `frame-src` rather than creating a second one, I'm not sure what the consequences of that might be 🤔.

Reply:
already merged but actioned in https://github.com/guardian/apps-rendering/pull/1251